projects/laji/src/app/shared-modules/select/select-subcategories/select-subcategories.component.html

```diff
@@ -0,0 +1,44 @@
+<div class="lj-select" [ngClass]="{'lj-close': open, 'lj-open': !open, 'lj-disabled': disabled}">
+  <strong *ngIf="title !== undefined && title !== ''" class="lj-title link" (click)="toggle($event, filter, typeCheckbox)">{{ title }}<laji-info *ngIf="info" [html]="info"></laji-info></strong>
+  <!-- GLOBAL FILTER -->
+  <laji-select
+  [options]="options"
+  [filterPlaceHolder]="'haseka.submissions.filter' | translate"
+  [useFilter]="false"
+  [selected]="selectedOptions['GLOBAL']"
+  [multiple]="true"
+  [info]="info"
+  [open]="open"
+  [disabled]="false"
+  [loading]="options['GLOBAL'] === null"
```

Comment:
This is used as and object here, but it's type in the `.ts` specified as `SelectOptions[]`. There is also quite a few errors in the console with text
```
ERROR TypeError: Cannot read property 'GLOBAL' of null